PIPs/pip-0005.md

```diff
@@ -0,0 +1,25 @@
+---
+title: address prefix
+status: Draft
+type: Standard
+author: Kayhan Alizadeh (kehiiiiya@gmail.com)
+created: 2023-08-27
+---
+
+## Abstract
+
+adding different prefix to different address types like: `validator` and `account` to have a more understandable address. 
+
+## Motivation
+
+address on `Pactus` have a `pc` prefix, for both validators and accounts. its impossible for humans and difficult for machine to find out address belongs to an `account`, `validator` or a `contract`. with adding suitable prefix we can find out the address role much faster and easier.
+
+## Specification
+
+to have suitable address prefix we can define `3` prefix for current types, `pv` for validators, `pc` for contracts and `pa` for account. all address must have correct prefix to be known as a valid address in `Pactus`.
+
+also the Possible future types must follow this rule and have their own prefix.
+
+## Copyright
+
+Copyright and related rights waived via [CC0](../LICENSE).
```

Comment:
@kehiy Don't remove `.md`

Reply:
with the `.md` pipeline throws errors!

Reply:
> with the `.md` pipeline throws errors!

If you put without `.md` link reference to branch LICENSE.

Reply:
I will add `.md`, but what should we do with the pipeline?